quinn-proto/src/connection/streams/recv.rs

```diff
@@ -107,7 +107,8 @@ impl Recv {
         // smaller than `stream_receive_window` in order to make sure the stream
         // does not get stuck.
         let diff = max_stream_data - self.sent_max_stream_data;
-        let transmit = self.receiving_unknown_size() && diff >= (stream_receive_window / 8);
+        let transmit =
+            self.receiving_unknown_size() && !self.stopped && diff >= (stream_receive_window / 8);
```

Comment:
There are four callers for `receiving_unknown_size()` and this commit adjusts three of them to consider `stopped` (the fourth is in `RecvStream::stop()` itself). Should we adjust the method itself and/or rename it and/or refactor it some other way?

Reply:
Renamed for clarity, then wrapped up the common `&& !stopped` pattern into a separate helper. We *could* rely exclusively on the new `Recv::can_send_flow_control`, even in `RecvStream::stop` with some careful reordering, but I think that's less clear, because `RecvStream::stop` specifically cares about whether *connection-level* flow control might need an update, judged based on whether the final offset is known, whereas all the other callsites care instead about whether *stream-level* flow control might need an update.